### ISSUE TITLE
Missing alt text: "who's using ember.js?" slider

### ISSUE THREAD

**Issue by MelSumner** [emberjs/website#3165]
**Context:** 
The alt text is missing for the slider that contains the logo images of companies who use Ember. This means that the link has no (machine) discernible text and would be meaningless to a user with assistive technology. 

**To Fix:**
Ensure that the company name is added as alt text in the template for the slider. 

**Screenshot:** 
![image](https://user-images.githubusercontent.com/4587451/35194595-8f1265ae-fe7b-11e7-8077-ef4c53f3623d.png)
